test/smoke/drive.js

```diff
@@ -23,11 +25,19 @@ describe('Drive', () => {
           return;
         }
 
-        this.slow(1000);
-
-        const { result: info } = await driveClient.request('getSyncInfo', {});
-
-        expect(info).to.have.property('lastSyncedBlockHeight');
+        this.timeout(100000);
+        this.slow(100000);
+
+        const time = Date.now() + 90000;
+        while (time > Date.now()) {
```

Comment:
Don't we use just for loop with a counter:
```
const timeout = 1000;
const attempts = 90;
for (const i = 0; i <= attempts; i++) {
     ...
     await wait(timeout);
}
```

Reply:
with for loop there potential problems:
1)requests inside loop could be proceed a long time. in this case 90 requests could take even hours
2) we are not able to track/calculate properly this.timeout and this.slow mocha methods